Skuid_Techniques/Multi_Part_Form/README.md

```diff
@@ -0,0 +1,33 @@
+# Multi-part Form Navigation and Real-time Completion Status
+
+Skuid can be used to create very effective complex data-entry forms.  There are two principles that can help users work through an extensive data-entry process:  
+- Break up the form into chunks.
+- Provide wayfinding and progress status information. 
+
+The example below uses the Skuid wizard component to break up a form into several different sections, but also uses the vertical navigation component - with selected-state conditions - to provide more effective wayfinding and status information for the whole form.  The example shows how to integrate the vertical navigation items and the wizard buttons so that both components stay in synch with user action. 
```

Comment:
```suggestion
The example below uses the Wizard component to break up a form into several different sections, but also uses the Vertical Navigation component - with selected-state conditions - to provide more effective wayfinding and status information for the whole form.  The example shows how to integrate the Vertical Navigation items and the Wizard buttons so that both components stay in sync with user action. 
```